spec/section/annotations.md

```diff
@@ -1,16 +1,294 @@
 ## Structural Annotations {#annotations}
 
-A [=field=] (`rml:field`) MUST contain zero or more [=structural annotations=]. 
-<dfn>Structural annotations</dfn> give additional information about the [=field=] (`rml:field`) and its relation to the structure of its [=logical view=] (`rml:LogicalView`).
-
-Following [=structural annotations=] MAY be defined: 
-- unique 
-- foreignKey
-- nonNullable
-- otherFunctionalDependency
-- iriSafe
-- (datatype also?)
+
+<s>A [=field=] (`rml:field`) contains zero or more [=structural annotations=].</s>
+
+<aside class="issue">
+Davide Lanti:
+
+I have a problem with definition above. While for certain annotations this might work (e.g., iriSafe and notNullable), for annotations possibly spanning across multiple fields (e.g., foreignKey and primaryKey) this won't suffice.
+
+My proposals:
+
+1) SQL way, having both variants: annotations for fields AND annotations for logical views;
+2) only having annotations at the level of the logical view
+
+In the remainder of this section, I try pursuing 1.
```

Comment:
I am in favor of option 2

Reply:
Ok. That is in fact how I wrote the section, in the end.